src/cheri_ptw.sail

```diff
@@ -24,22 +24,58 @@ function ptw_error_to_str(e) =
 
 overload to_str = {ptw_error_to_str}
 
+/*
+ * I apologize for the confusion you have experienced if you have skipped over
+ * this comment and read the body of this function first.  Hopefully these two
+ * points resolve most of it:
+ *
+ * 1) While it may look like we're signaling that we must *always* raise
+ * faults, translationException below can see whether the instruction
+ * triggering the PTW is claiming to store an asserted tag, and will ignore our
+ * result if that isn't so.
```

Comment:
We _are_ always raising faults? If we're not storing an asserted tag then we passed in `Data` so we'll `checkPTEPermission` will always say `PTW_SCM_OK`. And if we've been called then we know the PTE checking failed so we're definitely raising a (normal) page fault in that case.

Reply:
I have taken another stab at saying what I meant to be saying.